policies/aspnetcore.rego

```diff
@@ -0,0 +1,58 @@
+package tickets.aspnetcore
+
+import rego.v1
+
+default main := {"decision": false}
+
+auth_header := input.action.headers.Authorization
```

Comment:
[policy change]: The headers we get back from ASP.NET are capitalized by default (`Authorization` vs `authorization`)